main.go

```diff
@@ -118,24 +118,26 @@ func main() {
 		setupLog.Error(err, "Failed to get 'NAMESPACE' environment variable")
 		os.Exit(1)
 	}
+	availableCRDs := mapCRDAvailability(controllers.EgressFirewallCRD, controllers.EgressNetworkPolicyCRD, controllers.StorageClusterCRD, controllers.StorageClientCRD, controllers.OCSInitializationCRD)
```

Comment:
Why are we passing a list of CRD names here? I believe we agreed that the map will contain an existence mapping for all CRDs installed on the cluster.